frame/staking/src/lib.rs

```diff
@@ -1715,7 +1875,7 @@ impl<T: Trait> Convert<T::AccountId, Option<Exposure<T::AccountId, BalanceOf<T>>
 	for ExposureOf<T>
 {
 	fn convert(validator: T::AccountId) -> Option<Exposure<T::AccountId, BalanceOf<T>>> {
-		Some(<Module<T>>::stakers(&validator))
+		Some(<Module<T>>::eras_stakers(<Module<T>>::current_era(), &validator))
```

Comment:
shouldn't it be active_era instead of current era ?

it is used by historical session (at some point it shouldn't be necessary as historical session as all validator set exposure and know which one is the current one. But this can be done in a later PR).

historical session states:
> 	/// A conversion from validator ID to full identification.
	///
	/// This should contain any references to economic actors associated with the
	/// validator, since they may be outdated by the time this is queried from a
	/// historical trie.
	///
	/// It must return the identification for the current session index.
	type FullIdentificationOf: Convert<Self::ValidatorId, Option<Self::FullIdentification>>;